src/Polytopes/Polyhedron/constructors.jl

```diff
@@ -60,16 +56,21 @@ pm_polytope(P::Polyhedron) = P.pm_polytope
 
 ### Construct polyhedron from V-data, as the convex hull of points, rays and lineality.
 @doc Markdown.doc"""
-    convex_hull(V [, R [, L]])
+    convex_hull(V::Matrix [, R::Matrix [, L::Matrix]]; non_redundant::Bool = false)
 
 The polytope given as the convex hull of the rows of a set of points.
```

Comment:
This sentence makes no sense to me ... What are "rows of a set of points" ? Also, it seems to lack a verb... Perhaps "Return the polytope..." is meant?